dynawo/sources/Solvers/SolverSIM/DYNSolverSIM.cpp

```diff
@@ -133,8 +158,65 @@ SolverSIM::setSolverParameters() {
   nDeadband_ = findParameter("nDeadband").getValue<int>();
   maxRootRestart_ = findParameter("maxRootRestart").getValue<int>();
   maxNewtonTry_ =  findParameter("maxNewtonTry").getValue<int>();
-  linearSolverName_ = findParameter("linearSolverName").getValue<std::string>();
   recalculateStep_ = findParameter("recalculateStep").getValue<bool>();
+
+  linearSolverName_ = findParameter("linearSolverName").getValue<std::string>();
+  fnormtol_ = 1e-4;
```

Comment:
Look very similar to IDA, is there a way to factorize this?

Reply:
Done